docs/Developers/Review.rst

```diff
@@ -24,36 +60,33 @@ The review process for coala is as follows:
     - ``approved`` the commits have been reviewed by the developers and they
       are ready to be merged into the master branch
 
-5. The developers will acknowledge the commits by writing
-
-    - ``ack commit_SHA``, in case the commit is ready, or
-
-    - ``unack commit_SHA / commit_SHA needs work`` in case it is not ready yet
-      and needs some more work.
-
-6. If the commits are not linearly mergeable into master, rebase and go
+   If you don't have write access to coala, you can change the labels using
+   ``cobot mark wip <URL>`` or ``cobot mark pending <URL>``.
+3. The developers will acknowledge the commits by writing
+
+    - ``ack commit_SHA`` or ``commit_SHA is ready``, in case the commit is
+      ready, or
+    - ``unack commit_SHA`` or ``commit_SHA needs work`` in case it is not ready
+      yet and needs some more work or
+    - ``reack commit_SHA`` in case the commit was acknowledged before, was
+      rebased without conflicts and the rebase did not introduce logical
+      problems.
+4. If the commits are not linearly mergeable into master, rebase and go
    to step one.
-7. All commits are acknowledged and fit linearly onto master. All
+5. All commits are acknowledged and fit linearly onto master. All
    continuous integration services (as described below) pass. Anyone
    with collaborator permission may leave the ``@rultor merge`` command
```

Comment:
maybe tell them that a maintainer will do this? or we might get newcomers merging stuff